src/subscription_manager/gui/firstboot/rhsm_login.py

```diff
@@ -216,6 +217,9 @@ def shouldAppear(self):
         identity = inj.require(inj.IDENTITY)
         return not identity.is_valid()
 
+    def on_register_message(self, obj, msg, message_type=ga_Gtk.MessageType.ERROR):
```

Comment:
I believe ERROR is what the automation is expecting. Making the requested change now.